app/assets/javascripts/application.js

```diff
@@ -77,7 +77,7 @@ function onContentLoad(){
   $("#title_action a").addClass('btn').not('.btn-info, .btn-danger').addClass("btn-default");
 
   $("#title_action li a").removeClass("btn btn-default").addClass("la");
-  $("#title_action span").removeClass("btn btn-default").addClass("btn-group");
+  $("#title_action span").not('.glyphicon').removeClass("btn btn-default").addClass("btn-group");
```

Comment:
I think this change is no longer needed. 
As far as I can tell this whole line is not needed, but this can go to a separate PR.